automatic/thunderbird/tools/chocolateyInstall.ps1

```diff
@@ -50,6 +78,28 @@ if (!(Get-32bitOnlyInstalled($softwareName)) -and (Get-OSArchitectureWidth 64))
 }
 
 Install-ChocolateyPackage @packageArgs
+
+if ($pp.InstallDir) {
+  $installPath = $pp.InstallDir
+}
+else {
+  $installPath = Get-AppInstallLocation $softwareName
+}
+
+if (-Not(Test-Path ($installPath + "\distribution\policies.json") -ErrorAction SilentlyContinue) -and ($pp.NoAutoUpdate) ) {
+  if (-Not(Test-Path ($installPath + "\distribution") -ErrorAction SilentlyContinue)) {
+    new-item ($installPath + "\distribution") -itemtype directory
+  }
+
+  $policies = @{
+    policies = @{
+      "DisableAppUpdate" = $true
+    }
+  }
+  $policies | ConvertTo-Json | Out-File -FilePath ($installPath + "\distribution\policies.json") -Encoding ascii
```

Comment:
Unfortunately, the command `ConvertTo-Json` can not be used.

This cmdlet was added in PowerShell v3, and the packages in this repository and on the community repository is expected to be PowerShell v2 compatible.